wallet/udb/txdb.go

```diff
@@ -795,7 +799,7 @@ func putRawCredit(ns walletdb.ReadWriteBucket, k, v []byte) error {
 func putUnspentCredit(ns walletdb.ReadWriteBucket, cred *credit, scrType scriptType,
 	scrLoc uint32, scrLen uint32, account uint32) error {
 	k := keyCredit(&cred.outPoint.Hash, cred.outPoint.Index, &cred.block)
-	v := valueUnspentCredit(cred, scrType, scrLoc, scrLen, account)
+	v := valueUnspentCredit(cred, scrType, scrLoc, scrLen, account, DBVersion)
```

Comment:
All of the functions in this file should take the version as a parameter.  It's the store/manager (Store in this case) methods that should be hardcoding the parameter, not other key/value functions.